lib/sandboxed_renderer/init.js

```diff
@@ -78,6 +78,16 @@ if (window.location.protocol === 'chrome-devtools:') {
   require('../renderer/inspector')
 }
 
+if (binding.guestInstanceId) {
+  process.guestInstanceId = parseInt(binding.guestInstanceId)
+}
+
+if (!process.guestInstanceId && preloadProcess.argv.indexOf('--webview-tag=true') !== -1) {
```

Comment:
Can `process .guestInstanceId` be equal to `0` here?

Reply:
Guest always start from 1 and go higher.

https://github.com/electron/electron/blob/bcbcb4c6436e84e7f1f2387c2d7581bbdadb5732/lib/browser/guest-view-manager.js#L45-L65